antora-ui-camel/src/js/vendor/algoliasearch.bundle.js

```diff
@@ -4,8 +4,8 @@
   const algoliasearch = require('algoliasearch/lite')
 
   window.addEventListener('load', () => {
-    const client = algoliasearch('BH4D9OD16A', '16e3a9155a136e4962dc4c206f8278bd')
-    const index = client.initIndex('apache_camel')
+    const client = algoliasearch('QDOJ4LJQCC', '0fa28c3acc4d5f383b85ae4ad035ac9c')
+    const index = client.initIndex('aashna')
```

Comment:
The name should be `apache_camel`, I believe.

Reply:
As I mentioned, I've linked my personal index here, which is named "aashna". This is just for showing everyone how the search results will look with the new configuration. These two lines are not supposed to be merged, I will revert them before merging.